exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillPreparedStatementImpl.java

```diff
@@ -259,4 +261,17 @@ public void setObject(int parameterIndex, Object x, SQLType targetSqlType) throw
     checkOpen();
     super.setObject(parameterIndex, x, targetSqlType);
   }
+
+  @Override
+  public void setLargeMaxRows(long maxRowCount) throws SQLException {
+    Statement setMaxStmt = this.connection.createStatement();
+    setMaxStmt.execute("ALTER SESSION SET `" + ExecConstants.QUERY_MAX_ROWS + "`=" + maxRowCount);
+    setMaxStmt.close();
+    this.maxRowCount = maxRowCount;
+  }
+
+  @Override
+  public long getLargeMaxRows() {
```

Comment:
Please remove this method since Avatica provides its implementation.

Reply:
👍